src/ICeWall/doc/set_padring_options.md

```diff
@@ -9,7 +9,8 @@
       [-pad_inst_pattern pad_inst_pattern] \
       [-pad_pin_pattern pad_pin_pattern] \
       [-pin_layer pin_layer_name] \
-      [-connect_by_abutment signal_list]
+      [-connect_by_abutment signal_list] \
+      [-allow_filler_overlap|-reject_filler_overlap]
```

Comment:
I don't think you need both.  If the default is to reject then there isn't much purpose to the flag.  Compare with grt's -allow_overflow/-allow_congestion which don't have an opposite.

Reply:
Done